web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx

```diff
@@ -74,106 +121,268 @@ const FBCInputGrid = (props: FBCInputGridProps) => {
+            ...inputRow,
+            ...outputRow
+          }
+        ]
+      }
+      return []
     }
-    return column.render('Header')
-  }
+  ).flat()
+  const DECIMAL_PLACES = 1
+
   return (
-    <Table {...getTableProps()}>
-      <TableHead>
-        {headerGroups.map((headerGroup, hi) => (
-          <TableRow {...headerGroup.getHeaderGroupProps()} key={hi}>
-            {headerGroup.headers.map((column, hci) => (
-              <TableCell {...column.getHeaderProps()} key={hci}>
-                {renderHeader(column)}
-              </TableCell>
-            ))}
-          </TableRow>
-        ))}
-      </TableHead>
-      <TableBody>
-        {rows.map((row, ri) => {
-          prepareRow(row)
-          return (
-            <TableRow {...row.getRowProps()} key={ri}>
-              {row.cells.map((cell, ci) => {
+    <div className={classes.display} data-testid={props.testId}>
+      <Paper className={classes.paper} elevation={1}>
+        <TableContainer className={classes.tableContainer}>
+          <Table stickyHeader aria-label="daily table view of HFI by planning area">
+            <TableHead>
+              <TableRow>
+                <TableCell>
+                  <Checkbox
+                    color="primary"
+                    checked={headerSelected}
+                    onClick={() => {
+                      if (headerSelected) {
+                        // Toggle off
+                        updateSelected([])
+                        setHeaderSelect(false)
+                      } else {
+                        updateSelected(inputRows.map((_, i) => i))
+                        setHeaderSelect(true)
+                      }
+                    }}
+                  />
+                </TableCell>
+                <TableCell key="header-location">Weather Station</TableCell>
+                <TableCell key="header-elevation">
+                  Elev.
+                  <br />
+                  (m)
+                </TableCell>
+                <TableCell key="header-fuel-type">
+                  FBP
+                  <br />
+                  Fuel
+                  <br />
+                  Type
+                </TableCell>
+                <TableCell>
+                  Grass
+                  <br />
+                  Cure
+                  <br />
+                  (%)
+                </TableCell>
+                <TableCell>Status</TableCell>
+                <TableCell>
+                  Temp
+                  <br />
+                  (&deg;C)
+                </TableCell>
+                <TableCell>
+                  RH
+                  <br />
+                  (%)
+                </TableCell>
+                <TableCell>
+                  Wind
+                  <br />
+                  Dir
+                  <br />
+                  (&deg;)
+                </TableCell>
+                <TableCell className={classes.windSpeed}>
+                  {'Wind Speed (km/h) (Optional)'}
```

Comment:
do we still need the "(Optional)" part of the label anymore?

Reply:
Probably not, I can remove it